baseballAPI/ApiModels/PitchingStatsMapper.cs

```diff
@@ -171,133 +161,7 @@ private void ConvertOptionalParamsToNonNullable(Pitching pitching, PitchingStats
                 pitchingStats.Gidp = 0;
         }
 
-        private void ConvertOptionalLeaderboardParamsToNonNullable(Pitching pitching, PitchingLeaderBoardStats pitchingStats)
-        {
-            if (pitching.W != null)
```

Comment:
Leaving comment here, but taking about the function where these if/else checks still exist.  C# has a shortcut for this.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

So can be written something like:
`pitchingStats.W = pitching.W ?? 0;`

I think that works here, not sure if the types might not want to play correctly here.